drivers/SmartThings/philips-hue/src/handlers.lua

```diff
@@ -229,6 +226,7 @@ local function do_refresh_light(driver, light_device)
       else
         for _, light_info in ipairs(light_resp.data) do
           if light_info.id == light_resource_id then
+            light_device:set_field(Fields.COLOR_GAMUT, light_info.color.gamut_type, { persist = true })
```

Comment:
this doesn't have to be translated from JSON or anything?

Reply:
It's already translated by the time it gets here. The `api` module does that processing here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/ecfa1ffdb9c52a0a166c878492ec9f0f7e11e7c6/drivers/SmartThings/philips-hue/src/hue/api.lua#L57